plugins/opa/examples/dns-hostname-too-long/dns-hostname-too-long.yaml

```diff
@@ -0,0 +1,5 @@
+targets:
```

Comment:
This one might be a good one to do v2-style. Which (@ivanfetch-fw can correct me if I'm wrong) would just mean:
* removing this file
* adding `input.kind == "Ingress"` to the rego